scripts/docker-compose/docker-compose.private.yaml

```diff
@@ -0,0 +1,323 @@
+version: '3'
```

Comment:
what's the difference b/w public and private docker compose? do we need both? or is there any way to utilize the install.sh to change the variable values?